x/claim/abci.go

```diff
@@ -17,6 +17,6 @@ func EndBlocker(ctx sdk.Context, k keeper.Keeper) {
 	if goneTime > params.DurationUntilDecay+params.DurationOfDecay {
 		// airdrop time passed
 		k.FundRemainingsToCommunity(ctx)
-		k.ClearClaimables(ctx)
+		k.ClearInitialClaimables(ctx)
```

Comment:
I think we should refactor this into a single function called `endAirdrop()`, that then does both of these things internally.